packages/preview/starter-journal-article/0.1.1/typst.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "starter-journal-article"
+version = "0.1.1"
+entrypoint = "lib.typ"
+authors = ["HPDell <https://github.com/HPDell>"]
+license = "MIT"
+description = "A starter template for journal articles."
+repository = "https://github.com/HPDell/typst-starter-journal-article"
+exclude = ["*.pdf", "thumbnail.png"]
```

Comment:
You could ignore the .gitignore but that's not blocking

Reply:
Thanks. I've added `.gitignore` into `exclude`.